Xamarin.Forms.Core/AppTheme.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Xamarin.Forms
+{
+	public enum AppTheme
+	{
+		Unspecified,
+		Light,
+		Dark
```

Comment:
are we sure we won't need to extend this list ? if so, we might want to use strings...

Reply:
@StephaneDelcroix [Essentials defined it first in this way](https://docs.microsoft.com/en-us/xamarin/essentials/app-theme?context=xamarin%2Fxamarin-forms&tabs=android#obtaining-theme-information), so I think this was done to keep this in sync

Reply:
Indeed follow the Essentials way for now. I'd like to think this is like the OS level theming and on top of that people can have their own custom theming, probably as an extension of this (naming will be a hell...). But if you implement your own theme as a user, you still might want to also follow the light and dark mode by the OS and show different colors based on that.